packages/scene-composer/.eslintrc.js

```diff
@@ -56,5 +56,10 @@ module.exports = {
       },
     },
   ],
-  ignorePatterns: ['src/three/GLTFLoader.js', 'src/three/tiles3d/*', 'tools/watch-build.js'],
+  ignorePatterns: [
+    'src/three/GLTFLoader.js',
+    'src/three/tiles3d/*',
+    'tools/watch-build.js',
+    'src/assets/auto-gen/icons/*',
```

Comment:
do we have that linting yet? I ignore it here because the files there throw warning about missing return type. and since they are auto generated, we are not expected to manually add return types to them